packages/SidePanel/src/components/Content/Content.js

```diff
@@ -11,15 +11,17 @@ const defaultProps = {
   children: null,
 };
 
-export default function Content(props) {
+const Content = React.forwardRef((props, ref) => {
```

Comment:
It turns out I don't really need the ref on side panel content for the new component, but this change might be useful later on, so I didn't revert it.